content/blog/status-update-27-20.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "AsyncAPI Initiative Status Update (week 27, 2020)"
+date: 2020-06-30T06:00:00+01:00
+type: blog
+featured: true
+tags:
+  - Status
+  - Community
+#cover: /images/posts/asyncapi-conf-banner.webp
+weight: 10
+authors:
+  - name: Lukasz Gornicki
+    photo: /images/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+---
+
+## AsyncAPI is innovation and you should assess it
+
+ThoughtWorks and InfoQ clearly point out this is the best moment to have a closer look at AsyncAPI specification. Specification brings standardization into event-driven architectures and makes space for building great tools to solve problems like testing, documentation, code generation, and many others.
+
+Read more:
+- [ThoughtWorks Technology Radar - Tools](https://www.thoughtworks.com/radar/tools?blipid=202005046)
+- [Software Architecture and Design InfoQ Trends Report—April 2020](https://www.infoq.com/articles/architecture-trends-2020/)
+
+## Project Status Update
+
+Things we are focused on at the moment.
```

Comment:
This seems confusing to me. The whole blog post is about the status update. I'd just remove it and make sure all headings are correct.

Reply:
makes sense